.travis.yml

```diff
@@ -4,11 +4,14 @@ cache: bundler
 rvm:
   - 2.1.8
   - 2.2.4
-  - 2.3.0
+  - 2.3.1
```

Comment:
This is a much more used Ruby version I think. And it's what we use primarily right now.